selenium_tests/course_info/page_objects/course_people_page_object.py

```diff
@@ -40,41 +46,65 @@ def is_person_removed_from_list(self, lookup_text):
         user id
         """
         try:
-            WebDriverWait(self._driver, 30).until_not(lambda s: s.find_element(
+            WebDriverWait(self._driver, 60).until_not(lambda s: s.find_element(
```

Comment:
Is this the part that was failing? I'm not sure what you removed to fix the stale element exceptions.